db/migrate/20200906185544_add_delivery_method_to_distribution.rb

```diff
@@ -0,0 +1,5 @@
+class AddDeliveryMethodToDistribution < ActiveRecord::Migration[6.0]
+  def change
+    add_column :distributions, :delivery_method, :integer
```

Comment:
Could we instead use a `:string` instead of :integer? It could get difficult to understand what a 1 or 2 means when you are looking directly at the DB records.

Could we also default this to `pick-up`?

Reply:
> Could we also default this to pick-up?
When we talked about it last week, you said you thought we should leave it blank by default because we don't know what value existing distributions should have. By leaving it empty for now, it will require that any updates to existing deliveries will need to define the distribution method before changes can be saved.
But if you have changed your mind, we can default to pick-up.

Reply:
> Could we instead use a :string instead of :integer? It could get difficult to understand what a 1 or 2 means when you are looking directly at the DB records.

We could do that. But is human readability of DB records more important than following ActiveRecord best practices? Is looking directly at the DB records something normally done by our users?

Reply:
I don't think we should add this in this PR, but we could use https://github.com/alassek/activerecord-pg_enum to get the best of both worlds - efficient storage and better human readability in the DB

![0fc](https://user-images.githubusercontent.com/217050/93003588-d4f14280-f4f4-11ea-8fa9-e012d61b39b3.gif)

Reply:
Hey @daynew -- you are right! 

> But if you have changed your mind, we can default to pick-up by backfilling the existing distributions.

I totally forgot about mentioning that. Now that I'am thinking about it, I think we can default it to `pick-up` with the assumption that everything in the past was a `pick-up`. I think this ultimately makes them easier to manage since it doesn't need to consider `nil` (like in templates). For those reasons, I think we can do:
- Default to `pick-up`
- Set a not-null constraint at the DB level

> We could do that. But is human readability of DB records more important than following ActiveRecord best practices? Is looking directly at the DB records something normally done by our users? I would appreciate more context.

Good question. Currently, there are no users that are looking directly at the DB directly but it is something that I'am hoping to have in the future. However, since there is no use case right now of reading raw DB data, I think it's fine to keep it the way you have it.

> I don't think we should add this in this PR, but we could use https://github.com/alassek/activerecord-pg_enum to get the best of both worlds - efficient storage and better human readability in the DB

@albertchae nice! I didn't know about this. I think for now we can keep it as an integer. As mentioned above, there is no use case right now of directly using DB data.

Reply:
>
    Default to pick-up
    Set a not-null constraint at the DB level

Done